js/app/modules/contentGroup/contentGroup.js

```diff
@@ -166,9 +166,14 @@ const ContentGroup = new Module.Class({
 
     _on_models_changed: function () {
         let models = this._selection.get_models();
-        if (models.length === 0 && this._no_results) {
+        this.visible = true;
+        if (models.length === 0) {
             this._arrangement.set_models([]);
-            this._stack.visible_child_name = NO_RESULTS_PAGE_NAME;
+            if (this._no_results) {
```

Comment:
The set page shows two content groups, one for subsets of the current set, and one (below it) for articles of the current set. Some sets _only_ have subsets (and no articles) or _only_ have articles (and no subsets). In those cases you want the content group with no content in it to just hide